zulipterminal/helper.py

```diff
@@ -648,11 +648,7 @@ def check_narrow_and_notify(
 ) -> None:
     current_narrow = controller.model.narrow
 
-    if (
-        current_narrow != []
-        and current_narrow != outer_narrow
-        and current_narrow != inner_narrow
-    ):
+    if current_narrow != inner_narrow:
```

Comment:
This is the correct condition for the narrowing hint, but please see https://github.com/zulip/zulip-terminal/pull/1204#issuecomment-1100809732

There are two separate conditions; one handles the 'outside of narrow' part of the message (doesn't need updating), another separate one handles the showing of the second (hint) part.